src/lib/Signer.ts

```diff
@@ -0,0 +1,41 @@
+import type { WalletClient } from 'viem'
```

Comment:
I'm not sure if we want to add 'viem' as a dependency in the xmtp-react-native SDK. Just like how developers who use 'viem' wont want 'ethers' as a dependency, developers who use 'ethers' dont need to have 'viem' as a dependency either. 

I recognize that the `convertWalletClientToSigner` is a nice convenience feature, but it seems to go against the spirit of the PR.

Maybe we could remove the `viem` import in the SDK part of the code, and instead add a test to Example project that shows how you can create a Signer interface using a viem WalletClient?

Reply:
I see your point but consider that 'viem' is a dev dependency. it's only importing a type.

I won’t add more noise to the LaunchScreen but I'll try to do it as simply as possible. Thanks for your comment 😃

Reply:
Ah good point, you're right, if we're just importing types and adding as a dev dependency, it wont add to bundle size.  👍 In that case no update needed. Might just update the syntax on the other import so it also shows explicitly we're just importing the type. 👌